sql/custom/world/.gitignore

```diff
@@ -1 +0,0 @@
-*.sql
```

Comment:
you can keep this and still force adding the sql below. I would prefer to not have this .gitignore modified

Reply:
Thank you. I went ahead and re-included the gitignore.